mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraDeletedMessageDAO.java

```diff
@@ -123,12 +124,34 @@ public Mono<Void> addDeleted(CassandraId cassandraId, MessageUid uid) {
                 .setLong(UID, uid.asLong()));
     }
 
+    public Mono<Void> addDeleted(CassandraId cassandraId, Flux<MessageUid> uids) {
```

Comment:
We have a simpler reactive pipeline if we have a collection here. We should likely consider it.